_currencies.md

```diff
@@ -1,5 +1,117 @@
+
+## List Supported Assets
+
+> Example of unauthenticated request to list all supported assets:
+
+```bash
+curl https://api.uphold.com/v0/assets
+```
+
+> The above command returns JSON in the following format:
+
+```json
+[
+  {
+    "code": "BTC",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 8
+    },
+    "name": "Bitcoin",
+    "status": "open",
+    "symbol": "฿",
+    "type": "cryptocurrency"
+  },
+  {
+    "code": "USD",
+    "formatting": {
+      "decimal": ".",
+      "format": "__symbol__ __value__ __code__",
+      "grouping": ",",
+      "precision": 2
+    },
+    "name": "US Dollar",
+    "status": "open",
+    "symbol": "$",
+    "type": "fiat"
+  },
+  {
+    "code": "XPT",
+    "formatting": {
+      "decimal": ".",
+      "format": "__code__ __value__ __symbol__",
+      "grouping": ",",
+      "precision": 8
+    },
+    "name": "Platinum",
+    "status": "open",
+    "symbol": "oz",
+    "type": "commodity"
+  },
+  {
+    "code": "AAPL",
+    "formatting": {
+      "decimal": ".",
+      "format": "__value__ __code__",
+      "grouping": ",",
+      "precision": 6
+    },
+    "name": "Apple",
+    "status": "extended",
+    "type": "equity"
+  }
+]
+```
+
+> Example of authenticated request to list only the assets available to the current user:
+
+```bash
+curl https://api.uphold.com/v0/assets \
+  -H "Authorization: Bearer <token>"
+```
+
+Get the list of supported currencies and other financial assets, with details as described by the table below:
+
+Property   | Description
+---------- | --------------------------------------------------------------------------------------------------------
+code       | Uppercase abbreviation of the asset, e.g. "BTC", USD", "C", or "BRK.B".
+formatting | Specification for user-facing display, including number formatting and placement of the code and symbol.
+name       | Full name of the asset, e.g. "Euro", "Basic Attention Token", or "0x".
+status     | Current trading status. See [below](#asset-status) for more details.
+symbol     | A short and well-known representation of the asset, if one exists — e.g. "$", "฿", or "Kč".
+type       | Type of asset — one of `commodity`, `equity`, `cryptocurrency`, `fiat`, `stablecoin`, or `utility_token`.
+
+If the request is unauthenticated, the full list of assets supported by Uphold is returned.
+Authenticated requests, on the other hand, will filter the output, returning only the assets available for the current user, which can depend on factors such as their country and state.
```

Comment:
.. and state of residency

Reply:
[Fixed](https://github.com/uphold/docs/compare/4d82b9b..aac6359).